WcaOnRails/app/models/competition.rb

```diff
@@ -515,8 +521,12 @@ def has_entry_fee?
     base_entry_fee.nonzero?
   end
 
+  def has_competitor_limit?
+    competitor_limit_flag
```

Comment:
For option 1, if competitor_limit_enabled is false should we set competitor_limit to 0 in order to make it `not null`?